### PR TITLE
TWIS-38/avatar

### PR DESCRIPTION
**Pull request by Jack-Vu**
# TWIS-38/avatar

## Link to Story

https://re-boot.atlassian.net/browse/TWIS-38

## Description

Added a stateless avatar component that is has three fixed sizes: small 24x24px, medium 40x40px, and large 56x56px. Component is required to set if it is clickable or not and will handle a onClick call back function if it is able to be clicked. 

## Screenshot(s) / GIF(s):

image

image

image

image

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Dependency / Configuration update
- [ ] Documentation update

## Checklist:

- [x] My code follows the project's style guidelines
- [x] I have created a Storybook story for my component(s)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by javi-lopezz — APPROVED** [review]
LGTM, small,medium and large are all clickable when "isClickable" is true 

Screenshot 2024-03-08 at 10 35 21 AM

**Review by KrystalOm — COMMENTED** [review]
Screenshot 2024-03-08 at 11 03 04 AM
Small, medium, and large are clickable, but I don't get the pop-up window.

**Comment by Jack-Vu** [comment]
@KrystalOm to view the pop up you'll need to be in the Clickable story under the Avatar. You are currently in the Default which is doesn't handle the onClick function!

File: src/app/_components/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    case "small":
+      heightAndWidth = 24;
+      break;
+    case "medium":
+      heightAndWidth = 40;
+      break;
+    case "large":
+      heightAndWidth = 56;
+      break;
+  }
```
Comment (Muradil-Erkin) [inline src/app/_components/Avatar/Avatar.tsx:37]: this is fine, but an object mapping might be even better.

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]

**Comment by Muradil-Erkin** [comment]
@Jack-Vu Also, don't forget to include a `index.ts` to export the component to top level

**Review by HuiYangUS — APPROVED** [review]
> 1. Three sizes (small, medium, large): Matched
> 2. Sizes (24x24, 40x40, 56x56): Matched
> 3. Unclickable (Default Cursor=Arrow): Matched
> 4. Clickable (Cursor=Hand): Matched

* This PR is QA approved.

**Review by Muradil-Erkin — APPROVED** [review]

**Review by ShinK8245 — APPROVED** [review]
LGTM!

**Pull request merged**